pages/_app.js

```diff
@@ -22,6 +23,8 @@ function MyApp({ Component, pageProps, router }) {
         },
       }}
     >
+      <NextNProgress height={3} color="#fff" />
```

Comment:
The spinner at top right side looks bad. Is there a way to remove it?

Reply:
Yes we can, we just need to add this instead @niteshseram -

``` 
<NextNProgress height={3} color="#fff" options={{ showSpinner: false }} />
```

Reply:
So should I disable the spinner? @niteshseram

Reply:
@avneesh0612 Yes remove that.. as in it is hardly seen and when it is seen it is not serving much. IMO we can remove the spinner at the top right.

Reply:
okay @atapas removed it.